CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Feb X, 2025
+Include Vizro plotly templates after release of [vizro.bootstrap](https://github.com/mckinsey/vizro/pull/970)
+ - Add `vizro_dark.json` and `vizro.json` plotly templates
```

Comment:
In the changelog, let's include a link to the vizro github https://github.com/mckinsey/vizro/tree/main/vizro-core/src/vizro/_themes and say which version was used.